src/query/parser/promql/parse_test.go

```diff
@@ -76,6 +77,39 @@ func TestInvalidOffset(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestUnary(t *testing.T) {
+	q := "-up"
```

Comment:
worth adding `+up` case? Can probably generalize this to be like

```golang
for _, q := range []string{"+up","-up"} {
....
}
```

Reply:
`+` skips the unary op, so its just a fetch. Will add a test though.